frontend/src/components/ProjectCreate.vue

```diff
@@ -58,6 +59,29 @@
           </div>
         </div>
       </div>
+      <div v-if="state.project.tenantMode === 'TENANT'" class="col-span-1">
+        <label
+          class="text-lg leading-6 font-medium text-control select-none flex items-center"
+        >
+          {{ $t("project.db-name-template") }}
+          <BBCheckbox
+            :value="state.enableDbNameTemplate"
+            class="ml-2"
+            @toggle="(on: boolean) => state.enableDbNameTemplate = on"
+          />
+        </label>
+        <p class="mt-1 text-sm font-normal">
+          {{ $t("project.create-modal.db-name-template-tips") }}
+        </p>
+        <BBTextField
+          v-if="state.enableDbNameTemplate"
+          class="mt-4 w-full placeholder-gray-300"
+          :required="true"
+          :value="state.project.dbNameTemplate"
+          placeholder="{{DB_NAME}}_{{TENANT}}"
```

Comment:
placeholder="e.g. {{DB_NAME}}_{{TENANT}}"

otherwise, user may think that's the default value

Reply:
update, PTAL